home-manager/home-manager

```diff
@@ -263,8 +264,9 @@ function doBuild() {
     newsInfo=$(buildNews)
```

Comment:
The if here is required because the script tries to load the news file which will fail and the error is annoying.

Reply:
Hmm, I don't get any errors. Can you paste the error?

Reply:
``` 
+ /tmp/home-manager-build.xLOtx8D8bR/generation/activate
$HOME/.nix-profile/bin/home-manager: line 311: /tmp/home-manager-build.xLOtx8D8bR/generation/activate: No such file or directory
+ exitCode=1
```